admin/build-scientific-colors-cpt.sh

```diff
@@ -114,9 +121,9 @@ while read line; do
 	#
 	#	www.fabiocrameri.ch/visualisation
 	#
-	# License: Creative Commons Attribution 4.0 International License
-	# Copyright (c) 2020, Fabio Crameri All rights reserved.
-	# Crameri, F., (2020). Scientific colour maps. Zenodo. https://zenodo.org/record/4153113
+	# License: MIT License
```

Comment:
Seems like we've accidentally used the wrong license for a while :sweat_smile: The content of [Fabio's website](http://www.fabiocrameri.ch/colourmaps.php) is licensed under CC-BY-4.0 but the Scientific Colour Maps are under an MIT license according to http://www.fabiocrameri.ch/resources/ColourMaps/%2BREADME_ScientificColourMaps.pdf#1d.

Reply:
Thanks for noticing and correcting the problem